devicon.json

```diff
@@ -5489,6 +5489,42 @@
         "color": "#fed500",
         "aliases": []
     },
+    {
+        "name": "laminas",
+        "altnames": [
+            "Laminas Project",
+            "Zend Framework"
+        ],
+        "tags": [
+            "php",
+            "framework",
+            "web",
+            "mvc",
+            "middleware"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "original-wordmark",
+                "line",
+                "line-wordmark"
+            ],
+            "font": [
+                "original"
+            ]
+        },
+        "color": "#013755",
+        "aliases": [
```

Comment:
Avoid using aliases for existing SVG versions. For example, original and original-wordmark are distinct icons and already exist as SVGs.

Instead, use aliases for versions that aren't included in the `version.svg` or `version.font`. For instance, the plain version of an icon doesn't exist as an SVG, but since original icons are simple enough (They only have one fill color and one path) they can represent the plain versions of the icons. 

I recommend only including this alias:

    Base: original
    Alias: plain
    
    Base: original-wordmark
    Alias: plain-wordmark